readthedocs/oauth/services/bitbucket.py

```diff
@@ -181,19 +194,26 @@ def create_organization(self, fields):
         :rtype: RemoteOrganization
         """
         organization, _ = RemoteOrganization.objects.get_or_create(
-            slug=fields.get('username'),
-            account=self.account,
+            remote_id=fields['uuid'],
+            vcs_provider=self.vcs_provider_slug
         )
+        remote_organization_relation = self.get_remote_organization_relation(organization)
```

Comment:
It feels kind of magical that `self.account` is implicitly set on the relation in this function. I don't know if it's _bad_, but it feels a bit weird reading this code, and wondering how the account gets set on the relation.